### ISSUE TITLE
Accessability Violation : <ibm-panel> ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic" 

### ISSUE THREAD

**Issue by sant-santhiya** [carbon-design-system/carbon-components-angular#2519]
## Detailed description

> Describe in detail the issue you're having.

The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic". Need to remove 'aria-label' as it is not valid for div tag.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<ibm-header> ---> <ibm-panel> --> <div class="bx--panel--overlay" >`

> What did you expect to happen? What happened instead? What would you like to see changed?

Need to remove aria-label in div tag of `<ibm-panel>`

> What browser are you working in?

Mozila Firefox

> What version of the Carbon Design System are you using?

"carbon-components": "^10.56.0",
"carbon-components-angular": "^4.59.8",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

within one week

## Steps to reproduce the issue

Run the IBM Accessibility Checker tool on component. [ui-shell--right-panel-with-router](https://angular.carbondesignsystem.com/?path=/story/components-ui-shell--right-panel-with-router)

## Additional information

* Screenshots or code

![image](https://user-images.githubusercontent.com/104894503/231660280-bf57201b-0570-4b1d-8867-1d7033b1a515.png)

```
<div
    aria-label="Header panel"
    class="bx--panel--overlay bx--panel--expanded">
```

**Comment by makandre** [comment]
There's essentially the same issue reported on the react side, with more details and discussions https://github.com/carbon-design-system/carbon/issues/13464

**Comment by andyl309** [comment]
This issue is being flagged as a failure with our accessibility compliance and we will need to file a deviation without a fix. Does carbon team have resources to address this issue soon?

**Comment by keithcha** [comment]
It looks like removing [aria-label](https://github.com/carbon-design-system/carbon-components-angular/blob/4dcb72f3d34ee438a3134a489ae6c5bb1aaf21be/src/ui-shell/panel/panel.component.ts#L12C33-L12C33) could address this issue.
